<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferManagerDelegateWidget</class>
 <widget class="QWidget" name="TransferManagerDelegateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>64</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>64</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>64</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
  font-family: &quot;Lato&quot;;
}

TransferManagerDelegateWidget
{
    background-color: transparent;
}

#itemSepLine
{
    height: 1px;
    min-height: 1px;
    max-height: 1px;
    border: none;
    background: #EEEEEE;
	margin-left: 24px;
	margin-right: 24px;
}

#sStatus, #sStatus QWidget
{
    padding: 0;
    margin: 0;
}

#tFileType
{
    qProperty-iconSize: 48px;
    background: transparent;
}

QPushButton, QToolButton
{
    border: none;
    padding: 0;
    margin: 0;
}

QPushButton
{
    background: transparent;
}

#lTransferName
{
    color: #333333;
    font-size: 14px;
    qProperty-alignment: 'AlignLeft | AlignBottom';
}

QProgressBar
{
    border:0px;
    border-radius: 2px;
    background-color: #EEEEEE;
}

QProgressBar::chunk
{
  background-color: #6FD7FF;
  border-radius: 2px;
  border:0px;
}

#lDone, #lTotal
{
  font-size: 12px;
  font-weight: bold;
  padding: 0;
  margin: 0;
}

#lDone
{
  color: #999999;
  qProperty-alignment: 'AlignLeft';
}

#lTotal
{
  color: #333333;
 qProperty-alignment: 'AlignLeft';
}

#bItemSpeed
{
  color: #333333;
  font-size: 12px;
  font-weight: bold;
  text-align: left;
  qProperty-iconSize: 16px;
}

#sStatus
{
    width: 125px;
    min-width: 125px;
    max-width: 125px;
    height: 63px;
    min-height: 63px;
    max-height: 63px;
}

#lItemStatus, #lItemQueued, #lItemPaused, #lRetryMsg
{
	width: 125px;
    min-width: 125px;
    max-width: 125px;
    height: 63px;
    min-height: 63px;
    max-height: 63px;
}

#sStatus QLabel
{
    font-size: 12px;
    font-weight: bold;
    qProperty-alignment: 'AlignLeft | AlignVCenter';
    padding: 0;
    margin: 0;
}

#lItemStatus
{
  color: #00D26D;
}

#lItemQueued, #lItemPausedQueued_2
{
  color: #666666;
}

#lItemPaused,#lItemPausedQueued_1
{
  color: #FFA053;
}

#lItemFailed
{
  color: #F88183;
}

#lRetryMsg
{
  color: #F46265;
}

#lItemTime
{
    color: #333333;
    font-size: 12px;
    font-weight: bold;
    padding: 0;
    margin: 0;
}

#tItemRetry
{
    border-radius: 10px;
    background-color: #FFEDCC;
    color: #333333;
    font-size: 12px;
    font-weight: bold;
}

#tPauseResumeTransfer, #tCancelClearTransfer
{
    width: 16px;
    min-width: 16px;
    max-width: 16px;
    height: 16px;
    min-height: 16px;
    max-height: 16px;
    qProperty-iconSize: 16px;
    padding-left: 0px;
    margin-right: 0px;
    border: none;}</string>
  </property>
  <layout class="QVBoxLayout" name="TransferManagerItemLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Line" name="itemSepLine">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="sTransferState">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>63</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>63</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stateActive">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>702</width>
        <height>63</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>63</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="stateActiveLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>24</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="tFileType">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>8</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="wName" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>105</width>
           <height>63</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>63</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="wNameLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="lTransferName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>155</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wProgressBar" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>38</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QProgressBar" name="pbTransfer">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>4</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>4</height>
                </size>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="format">
                <string notr="true"/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wSize" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>63</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>63</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QLabel" name="lDone">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignVCenter">
           <widget class="QLabel" name="lTotal">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bItemSpeed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>63</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>63</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="sStatus">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>63</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>63</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>5</number>
         </property>
         <widget class="QWidget" name="pActive">
          <widget class="QLabel" name="lItemStatus">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>125</width>
             <height>63</height>
            </rect>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pQueued">
          <widget class="QLabel" name="lItemQueued">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>125</width>
             <height>63</height>
            </rect>
           </property>
           <property name="text">
            <string>Queued</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pRetry">
          <widget class="QLabel" name="lRetryMsg">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>125</width>
             <height>63</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pPaused">
          <widget class="QLabel" name="lItemPaused">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>125</width>
             <height>63</height>
            </rect>
           </property>
           <property name="text">
            <string>Paused</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pPausedQueued">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lItemPausedQueued_1">
             <property name="text">
              <string>Paused</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lItemPausedQueued_2">
             <property name="text">
              <string>(queued)</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pFailed">
          <layout class="QHBoxLayout" name="pFailedLayout" stretch="0,1,0">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lItemFailed">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>63</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>63</height>
              </size>
             </property>
             <property name="text">
              <string>Failed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tItemRetry">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Retry</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lItemTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>63</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>63</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QWidget" name="wPauseResume" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>63</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>63</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="wPauseResumeLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="tPauseResumeTransfer">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="icon">
             <iconset resource="../../../gui/Resources_linux.qrc">
              <normaloff>:/images/transfer_manager/transfers_actions/lists_pause_ico_default.png</normaloff>:/images/transfer_manager/transfers_actions/lists_pause_ico_default.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QWidget" name="wClearCancel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>63</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>63</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="wClearCancelLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="tCancelClearTransfer">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="icon">
             <iconset resource="../../../gui/Resources_linux.qrc">
              <normaloff>:/images/transfer_manager/transfers_actions/lists_cancel_ico_default.png</normaloff>:/images/transfer_manager/transfers_actions/lists_cancel_ico_default.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../gui/Resources_linux.qrc"/>
 </resources>
 <connections/>
</ui>
